src/GenFu/Fillers/DateTimeNullableFillers.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace GenFu
+{
+    public class DateTimeNullableFiller : PropertyFiller<DateTime?>
```

Comment:
I disagree, I just fixed a bug in DateTime fillter which also appears in this code. Had the two shared the code then the bug wouldn't appear in this filler. I do agree that inherit is too strong of a relationship but you could certainly use an adapter pattern to wrap an instance of DateTimeFiller.